requirements.txt

```diff
@@ -3,9 +3,15 @@ Django==1.10.3
 django-cors-middleware==1.3.1
 django-filter==1.0.0
 djangorestframework==3.5.3
+factory-boy==2.8.1
+Faker==0.7.7
 google-api-python-client==1.5.5
 httplib2==0.9.2
 oauth2client==4.0.0
 oauthlib==2.0.0
+pyasn1==0.1.9
```

Comment:
what keeps adding pyasn.... O_O

(from what I can tell nothing actually *needs* it, it just gets added for... convenience?)